### PR TITLE
Dac 1832 expose get disk space info from birdge

### PR DESCRIPTION
**Pull request by andyindahouse**

### CODE REVIEW

File: src/post-message.ts [hunk 1]
```diff
@@ -202,14 +207,14 @@ const hasWebKitPostMessage = () =>
 const getWebViewPostMessage = (): NovumPostMessage | null => {
     // Android
     if (hasAndroidPostMessage()) {
-        return (jsonMessage) => {
+        return jsonMessage => {
```
Comment (atabel) [inline src/post-message.ts:210]: different prettier versions?
don't we have prettier check in ci?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v.2.17.0 - 2020-06-17
```
Comment (atabel) [inline CHANGELOG.md:3]: don't edit the changelog. You don't know when the new version will be published

File: src/post-message.ts (hunk 1 above)
Comment (andyindahouse) [reply src/post-message.ts:210]: yes, I don't know why vscode take a different version of prettier when I save changes. I had to run prettier with --write to fix this

File: CHANGELOG.md (hunk 2 above)
Comment (andyindahouse) [reply CHANGELOG.md:3]: okey

File: src/post-message.ts (hunk 1 above)
Comment (andyindahouse) [reply src/post-message.ts:210]: Anyway I think that the husky config doesn't work well else I couldn't commit these changes

**Comment by atabel** [comment]
the windows build is failing

File: README.md [hunk 3]
```diff
@@ -619,6 +620,27 @@ setCustomerHash(
 );
 ```
 
+### getDiskSpaceInfo
+
+Return info about how much free disk space there is in the device
```
Comment (sergivillar) [inline README.md:625]: this phrase sounds weird to me  xDD
'how much free disk space there is in the device' is it correct? just talking about english xDD
how much free disk space the device has?

File: README.md [hunk 4]
```diff
@@ -619,6 +620,27 @@ setCustomerHash(
...
+#### Example
+
+```javascript
+import {getDiskSpaceInfo} from '@tef-novum/webview-bridge';
+
+getDiskSpaceInfo().then(({availableBytes, totalBytes}) => { ... });
+```
+
+-   `availableBytes`: number to see available bytes in the device
+-   `totalBytes`: number to see the total bytes available in the device
```
Comment (sergivillar) [inline README.md:642]: I'll remove the "available" part of this description to make it less confuse with de previous one

File: README.md (hunk 3 above)
Comment (andyindahouse) [reply README.md:625]: lol, yes, you're right, I'll change this

File: README.md (hunk 4 above)
Comment (andyindahouse) [reply README.md:642]: 👍

**Pull request merged**